src/MQTTClient.php

```diff
@@ -1313,7 +1341,7 @@ protected function writeToSocket(string $data, int $length = null): void
 
         $length = min($length, strlen($data));
 
-        $result = @fwrite($this->socket, $data, $length);
+        $result = fwrite($this->socket, $data, $length);
```

Comment:
Not happy with this, for two reasons:

1. This changeset is mostly doesn't have any functional change at all, besides the text strings. Not good to bury something with potentially a big impact like this in such a changeset, rather have it on a separate commit (but see number 2).

2. Unfortunately, in PHP you *have to* use the `@` operator for fwrite() because the socket might abruptly fail in that very moment, before you can catch the feof(), and PHP will start shouting, which I find very irritating and forces to handle PHP errors externally, I usually prefer to have them on my stderr to point out real bugs, while a failing socket is something to be expected and handled. It is the same old story of `@fopen()`